lambda/aws-rag-appsync-stepfn-kendra/kendra_job_manager/update_job_status.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import boto3
+import json
+import os
+import re
+import datetime
+from datetime import datetime, timezone
+import uuid
+
+ddb_client = boto3.client('dynamodb')
+DOCUMENTS_TABLE = os.environ["DOCUMENTS_TABLE"]
+dynamodb_client = boto3.resource('dynamodb')
+
+def lambda_handler(event, context):
+    table = dynamodb_client.Table(DOCUMENTS_TABLE)
+    table.update_item(
```

Comment:
what if this fails ? https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/update_table.html section Exceptions